.github/workflows/wolfi-presubmit.yaml

```diff
@@ -108,10 +108,13 @@ jobs:
         run: |
           for f in packages/x86_64/${{ matrix.package }}-*.apk; do
             docker run --rm -v $(pwd):/work cgr.dev/chainguard/wolfi-base apk add --allow-untrusted /work/$f
-            wolfictl scan \
```

Comment:
Oh if one can scan all packages in one go, then for loop should be removed altogether no?

Reply:
I don't think they can be _installed_ in one go safely.

Reply:
Let's try it.

Reply:
@xnox Do we need scanning here in the first place? I'm guessing this was added for a reason, just not sure why yet

Reply:
Nice it worked!

Reply:
@luhring yes to ensure that any changes of melange pipelines do not break scanning the binaries. Like when i changed how SBOMs look and what they contain. Because in theory any changes of melange, should still produce scanable / parsable binaries, without any change of CVE detection. As discussed on when SBOM changes with external references got added.